catkin_tools/jobs/utils.py

```diff
@@ -25,6 +25,20 @@
 from catkin_tools.execution.events import ExecutionEvent
 
 
+class CommandMissing(Exception):
+    '''A required command is missing.'''
+
+    def __init__(self, name):
+        super(CommandMissing, self).__init__(
+                'Cannot find required tool `%s` on the PATH, is it installed?' % name)
+
+
+def require_command(name, which):
+    if not which:
+        import errno
```

Comment:
The function is simple enough now that it could probably just be inlined; I don't really care one way or another about that.